### DISCUSSION TITLE
Run pyfluent for 2D solver

### DISCUSSION THREAD

**Discussion by quiquedios** [ansys/pyfluent#1910] [Q&A]
Hello,

I am wondering if there is any chance that pyfluent works in 2D, which I guess it does, but how can I activate this mode? By default, my solver starts in 3D so I get the following error when trying to import my 2D mesh:
**********************
_Multicore SMT processors detected. Processor affinity set!
Reading "D:\MESH.msh"...
Buffering for file scan..._

_Error: File has wrong dimensions (2).
Error Object: #f_

_Error: Error reading "D:\MESH.msh".
Error Object: #f_
**********************
These are the commands I ran:
***********************
_import ansys.fluent.core as pyfluent
solver = pyfluent.launch_fluent(precision="double", processor_count=4, mode="solver", )
solver.health_check_service.is_serving_
--> True
_solver.file.read_mesh(file_name = 'D:\MESH.msh')_
***************************
I apologise as I am new to pyfluent.

Thanks,

Enrique
**********************************
**EDIT: Already solved**

**Comment by mkundu1** [comment]
```
solver = pyfluent.launch_fluent(version="2d", precision="double", processor_count=4)
```

**Reply by LSCHH** [reply]
maybe should dimension=2 now
